graph/src/env/mod.rs

```diff
@@ -326,6 +330,8 @@ struct Inner {
     elastic_search_flush_interval_in_secs: u64,
     #[envconfig(from = "GRAPH_ELASTIC_SEARCH_MAX_RETRIES", default = "5")]
     elastic_search_max_retries: usize,
+    #[envconfig(from = "GRAPH_ELASTIC_SEARCH_INDEX", default = "subgraph-logs")]
```

Comment:
can we make the alias the default - `subgraph`

Reply:
Just changed that